test-clients/src/main/java/com/hedera/services/bdd/spec/queries/schedule/HapiGetScheduleInfo.java

```diff
@@ -0,0 +1,156 @@
+package com.hedera.services.bdd.spec.queries.schedule;
+
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.spec.infrastructure.HapiSpecRegistry;
+import com.hedera.services.bdd.spec.queries.HapiQueryOp;
+import com.hedera.services.bdd.spec.queries.token.HapiGetTokenInfo;
+import com.hedera.services.bdd.spec.transactions.TxnUtils;
+import com.hederahashgraph.api.proto.java.HederaFunctionality;
+import com.hederahashgraph.api.proto.java.Query;
+import com.hederahashgraph.api.proto.java.Response;
+import com.hederahashgraph.api.proto.java.ScheduleGetInfoQuery;
+import com.hederahashgraph.api.proto.java.Timestamp;
+import com.hederahashgraph.api.proto.java.Transaction;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+
+import java.util.Optional;
+import java.util.function.BiFunction;
+
+import static com.hedera.services.bdd.spec.queries.QueryUtils.answerCostHeader;
+import static com.hedera.services.bdd.spec.queries.QueryUtils.answerHeader;
+
+public class HapiGetScheduleInfo extends HapiQueryOp<HapiGetScheduleInfo> {
+    private static final Logger log = LogManager.getLogger(HapiGetTokenInfo.class);
+
+    String schedule;
+
+    public HapiGetScheduleInfo(String schedule) {
+        this.schedule = schedule;
+    }
+
+    Optional<String> expectedScheduleId = Optional.empty();
+    Optional<String> expectedSchedulingAccount = Optional.empty();
+    Optional<String> expectedPayer = Optional.empty();
+    Optional<byte[]> expectedTransactionBody = Optional.empty();
+    Optional<Boolean> expectedExecuteImmediately = Optional.empty();
+    Optional<Timestamp> expectedExecuteAtTime = Optional.empty();
```

Comment:
lets be consistent with the Protobufs and state and rename the `expectedSchedulingAccount` and `expectedPayer`. We should delete `expectedExecuteImmediately` and `expectedExecuteAtTime` as-well

We should add `adminKey` and `signers` as-well